modules/op-starting-pipelines.adoc

```diff
@@ -3,66 +3,68 @@
 // *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
 
 [id="op-starting-pipelines_{context}"]
-= Starting Pipelines
+= Starting pipelines
 
-After you create a Pipeline, you need to start it  to execute the included Tasks in the defined sequence. You can start a Pipeline Run from the *Pipelines* view, *Pipeline Details* page, or the *Topology* view.
+After you create a pipeline, you need to start it to execute the included tasks in the defined sequence. You can start a pipeline from the *Pipelines* view, the *Pipeline Details* page, or the *Topology* view.
 
 .Procedure
-To start a Pipeline using the *Pipelines* view:
+To start a pipeline using the *Pipelines* view:
 
-. In the *Pipelines* view of the *Developer* perspective, click the *Options* {Kebab} menu adjoining a Pipeline, and select *Start*.
-. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the Pipeline definition.
+. In the *Pipelines* view of the *Developer* perspective, click the *Options* {Kebab} menu adjoining a pipeline, and select *Start*.
+. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the pipeline definition.
 +
 [NOTE]
 ====
-For Pipelines created using the *From Git* option, the *Start Pipeline* dialog box also displays an `APP_NAME` field in the *Parameters* section, and all the fields in the dialog box are prepopulated by the Pipeline templates.
+For pipelines created using the *From Git* option, the *Start Pipeline* dialog box also displays an `APP_NAME` field in the *Parameters* section, and all the fields in the dialog box are prepopulated by the pipeline template.
 ====
 +
-.. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the Pipeline Run instance.
-. Optional: Modify the *Advanced Options* to add credentials to authenticate the specified private Git server or Docker registry.
+.. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the pipeline run instance.
+. Optional: Modify the *Advanced Options* to add credentials that authenticates the specified private Git server or the Docker registry.
```

Comment:
Should "to add credentials that authenticates" be "to add the credentials that authenticate"?